src/test/groovy/org/prebid/server/functional/model/config/Stage.groovy

```diff
@@ -12,7 +12,8 @@ enum Stage {
     BIDDER_REQUEST("bidder-request"),
     RAW_BIDDER_RESPONSE("raw-bidder-response"),
     PROCESSED_BIDDER_RESPONSE("processed-bidder-response"),
-    AUCTION_RESPONSE("auction-response")
+    AUCTION_RESPONSE("auction-response"),
+    ALL_PROCESSED_BID_RESPONSES("all-processed-bid-responses")
```

Comment:
I'll leave a separate comment again. Please arrange those in the order occur in the PBS.

Reply:
updated